ReactWindows/ReactNative/Touch/TouchHandler.cs

```diff
@@ -77,7 +77,9 @@ private void OnPointerPressed(object sender, PointerRoutedEventArgs e)
 
             var originalSource = e.OriginalSource as DependencyObject;
             var rootPoint = e.GetCurrentPoint(_view);
-            var reactView = GetReactViewTarget(originalSource, rootPoint.Position);
+            var transform = (sender as UIElement).TransformToVisual(Window.Current.Content);
```

Comment:
It's not great practice to use `TypeAs` expressions (i.e., `sender as UIElement`) without a null check. It should be safe to use an explicit cast here (i.e., `((UIElement)sender)`), since you know the sender will always be the `ReactRootView`.